src/App.jsx

```diff
@@ -1,10 +1,24 @@
 import { useState } from 'react'
 import './App.css'
+import './stylesheets/movielist.css'
```

Comment:
Overall, the components should be isolated and reusable pieces of code, they don't need to have any insight of where they're placed. I would recommend putting the container inside the MovieList component itself, and here in the App component just add any css to position the MovieList on the page, let's say that if the App wants to position it at 100vw and 100vh, the MovieList doesn't care, if suddenly it wants to put it at the left, MovieList is not aware of that, same to any other change.
![image](https://github.com/user-attachments/assets/d9d183c7-321c-49e8-af12-fe31e661a55c)